File: Uri-iniciante/1008.cpp
#include<stdio.h>
/*	
*	-----------------------------------
*	|		Pedro Daniel Jardim       |
*	| 			  UFV				  | 
*	|		  25/02/2020			  |
*   ----------------------------------
*
*/



int main() 
{
		int number,hours;
		float salary_hours;
		
		scanf("%d %d",&number,&hours);
		scanf("%f",&salary_hours);
		salary_hours=salary_hours*hours;
		
		printf("NUMBER = %d\n",number);
		printf("SALARY = U$%.2f ",salary_hours);
 

   return 0;
   
}

File: Uri-Iniciante/1008.java
import java.io.IOException;
import java.util.Scanner;
/*
 * 
*/
public class Main {
	public static void main(String[] args) throws IOException {
		Scanner sc = new Scanner(System.in);
		int number,hours;
		double salary_hours;
		
		number = sc.nextInt();
		hours= sc.nextInt();
		salary_hours = sc.nextDouble();
		salary_hours=salary_hours*hours;
		
		System.out.println("NUMBER = "+number);
		System.out.println("SALARY = U$ "+String.format("%.2f",salary_hours));

		sc.close();
	}

}